### ISSUE TITLE
Key "from" in data for Cloud Messaging can't be used.

### ISSUE THREAD

**Issue by abdielcs** [kreait/firebase-php#574]
PHP 7.3.24

kreait/clock                            1.1.0     A PHP 7.0 compatible clo...
kreait/firebase-bundle                  2.3.0     Symfony Bundle for the F...
kreait/firebase-php                     5.15.0    Firebase Admin SDK
kreait/firebase-tokens                  1.14.0    A library to work with F...

At least sending the notification to multiple devices by token, the key "from" can't be used in data. `Invalid data payload key: from`. A warning would have been helpful.

```
$message = CloudMessage::new()
            ->withNotification($notification)
            ->withData(['from' => 'any']);
```

**Comment by jeromegamez** [comment]
Thanks for the report! https://firebase.google.com/docs/cloud-messaging/concept-options has a list of reserved keys for the data payload. I'll add a check in the SDK, please be aware that the SDK doesn't emit warnings - once implemented, this will throw an exception when one of the reserved key names is used.

**Comment by jeromegamez** [comment]
Implemented with d7fca18a3639d8898dcb6385ecf6d97df349d237, I want to look into another issue before I release it, but you can expect it in the next few days 🤞🏼. Thanks!
